lldb/source/API/CMakeLists.txt

```diff
@@ -40,6 +40,113 @@ add_custom_target(lldb-sbapi-dwarf-enums
   DEPENDS ${sb_languages_file})
 set_target_properties(lldb-sbapi-dwarf-enums PROPERTIES FOLDER "LLDB/Tablegenning")
 
+if(CMAKE_SYSTEM_NAME MATCHES "AIX")
+add_lldb_library(liblldb STATIC ${option_framework}
```

Comment:
I'm sure we can find a way to refactor this, deal with that later, but is there a reason that it needs to be static? Is this a property of libraries on AIX perhaps.

I know we had someone wanting to build a static library elsewhere but I'm not sure if it was this one.